docs/sources/architecture/ring.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Ring
+weight: 1000
+---
+
+# Ring
+
+This document builds upon the information in the [Loki Architecture](./) page
+
+It explains what exactly is the ring, what problem does it solve and how it actually works.
+
+We use ring in Distributor and Ingester components of Loki.
+
+Loki in microservice mode usually can have multiple ingesters and distributors. This multiple instances of same component(ingester or distributor) forms a ring (more presicisly Consisten Hash Ring).
+
+Both distributors and ingesters have their own ring. Write path looks like Client -> Distributors (ring) -> Ingesters (ring). Read path looks like Client -> Querier -> Ingesters (ring).
```

Comment:
Not sure what do you mean by.

> Both distributors and ingesters have their own ring.

Reply:
I actually mean two separate rings for distributor and ingestor.

distributor - https://github.com/grafana/deployment_tools/pull/8186
ingestor - https://github.com/grafana/loki/blob/master/pkg/ingester/ingester.go#L52

Reply:
I see but there's not all hashring. It seems that you explain hashring here. At least I would focus on ring type to avoid confusion.